flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/ChangelogAggregatingState.java

```diff
@@ -67,18 +73,24 @@ public OUT get() throws Exception {
     @Override
     public void add(IN value) throws Exception {
         delegatedState.add(value);
+        changeLogger.valueUpdated(delegatedState.getInternal(), getCurrentNamespace());
```

Comment:
The same as `ChangelogReducingState`, update changelog first and then add to delegated state.